DependencyInjection/Configuration.php

```diff
@@ -49,6 +49,9 @@ public function getConfigTreeBuilder()
                                 ->booleanNode('guess_uncaught_frames')
                                     ->defaultFalse()
                                 ->end()
+                                ->scalarNode('rollbarjs_version')
+                                    ->defaultValue('')
```

Comment:
I think it would be better to setup the default version here, instead `RollbarExtension::getRollbarJsVersion`
(https://github.com/ftrrtf/FtrrtfRollbarBundle/pull/16/files#diff-38c58ba1801f78ea80e449d3547809dbR109)